app/presenters/idv/in_person/usps_states_territories_presenter.rb

```diff
@@ -0,0 +1,8 @@
+include FormHelper
+
+class Idv::InPerson::UspsStatesTerritoriesPresenter
+  # Filtered list to remove territories that cause errors in the USPS API
+  def usps_states_territories
+    FormHelper.us_states_territories.reject { |_name, abbrev| %w[AA AE AP UM].include?(abbrev) }
```

Comment:
Hm, I hadn't considered the use of a helper in a presenter class like this. I wonder if it would work to do this? Slightly simpler, and avoids the `include` outside the class itself.

```suggestion
class Idv::InPerson::UspsStatesTerritoriesPresenter
  include FormHelper
  
  # Filtered list to remove territories that cause errors in the USPS API
  def usps_states_territories
    us_states_territories.reject { |_name, abbrev| %w[AA AE AP UM].include?(abbrev) }
```

Alternatively, we could use the states at [`Idp::Constants::STATE_AND_TERRITORY_CODES`](https://github.com/18F/identity-idp/blob/061d6338bf47d3350c331ed0d0dcaaa12f28c4cb/lib/idp/constants.rb#L20-L81)? I'm not sure why they're repeated, and this makes me think we should maybe do away with `us_states_territories` altogether as a separate effort (use `STATE_AND_TERRITORY_CODES` + locale strings instead).

Reply:
Looks like this works. Will commit it!

Looks like STATE_AND_TERRITORY_CODES doesnt even include the removed items. (Armed Forces Territories and Minor islands). Also if we get rid of us_states_territories we would still need some structure with the full state/territory names mapped to codes for the select items.

Reply:
The only place the old list is being used is `app/views/idv/address/new.html.erb `

Reply:
The thought with using `STATE_AND_TERRITORY_CODES` is that `us_states_territories` includes hard-coded state names in English, which means they're not translated when shown in other languages. Using `STATE_AND_TERRITORY_CODES`, we could use our normal translation tools to have something like...

```
# config/locales/en.yml
states.az: Arizona
# ...
```

```rb
# app/presenters/idv/in_person/usps_states_territories_presenter.rb

  def usps_states_territories
    STATE_AND_TERRITORY_CODES.map do |abbrev|
      [abbrev, I18n.t(abbrev, scope: [:states])]
    end
  end
```
```